microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -149,21 +153,24 @@ Passwordless authentication is a method of verifying a user’s identity without
 Microsoft Graph PowerShell supports the following passwordless authentication methods:
 
 - Windows Hello for Business
-- Fast ID Online v2.0 (FIDO2)
+- Fast Identity Online v2.0 (FIDO2)
 - Microsoft Authenticator app
 - Certificate-based authentication (CBA)
 
 >[!NOTE]
-> FIDO2 security keys option is only supported on PowerShell 7 and above.
+> The use of FIDO2 security keys for authentication is supported only in PowerShell 7 and above.
 
-For more information, see [Passwordless authentication options for Microsoft Entra ID](/azure/active-directory/authentication/concept-authentication-passwordless) and [Microsoft Entra certificate-based authentication](/azure/active-directory/authentication/concept-certificate-based-authentication).
+For more information, see [Passwordless authentication options for Microsoft Entra ID](/azure/active-directory/authentication/concept-authentication-passwordless) and [Microsoft Entra ID certificate-based authentication](/azure/active-directory/authentication/concept-certificate-based-authentication).
 
 ## Use Disconnect-MgGraph
 
-Once you're signed in, you'll remain signed in until you invoke `Disconnect-MgGraph`. Microsoft Graph PowerShell automatically refreshes the access token for you and sign-in persists across PowerShell sessions because Microsoft Graph PowerShell securely caches the token.
+Once you're signed in, you'll remain signed in until you invoke `Disconnect-MgGraph`. Microsoft Graph PowerShell automatically refreshes the access token for you, and sign-in persists across PowerShell sessions because Microsoft Graph PowerShell securely caches the token when using the default `CurrentUser` context scope. If you use the `-ContextScope Process` parameter with `Connect-MgGraph`, sign-in will only persist for the current PowerShell session.
```

Comment:
```suggestion
Once you're signed in, you remain signed in until you invoke `Disconnect-MgGraph`. Microsoft Graph PowerShell automatically refreshes the access token for you, and sign-in persists across PowerShell sessions because Microsoft Graph PowerShell securely caches the token when using the default `CurrentUser` context scope. If you use the `-ContextScope Process` parameter with `Connect-MgGraph`, sign-in only persists for the current PowerShell session.
```